src/Domain/Donation.php

```diff
@@ -215,14 +215,19 @@ public function toHookModel(): array
         $data['updatedTime'] = $this->getUpdatedDate()->format(DateTime::ATOM);
         $data['amountMatchedByChampionFunds'] = (float) $this->getConfirmedChampionWithdrawalTotal();
         $data['amountMatchedByPledges'] = (float) $this->getConfirmedPledgeWithdrawalTotal();
+        $data['optInCharityEmail'] = $this->getCharityComms();
+
+        // For backwards compatability with Enthuse
+        if ($this->getPsp() != 'stripe') {
```

Comment:
@NoelLH would you prefer strict `!==` here?

Reply:
@tbg-jasonfung yes, comparisons should always be strict unless there is a reason not.

Moreover, why don't we set the value only when Stripe is involved rather than setting then un-setting? And is there a reason to base it specifically on PSP rather than whether a value is present?

Reply:
@NoelLH Okay thanks - yeah think it makes sense to set the value when present rather than checking for psp, i've made a change to reflect this